samples/bluetooth/mesh/boards/bbc_microbit.conf

```diff
@@ -7,6 +7,8 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
 CONFIG_MICROBIT_DISPLAY_STR_MAX=5
 CONFIG_GPIO=y
 
+CONFIG_HWINFO=n
```

Comment:
This was not in the original file, is it needed?

Reply:
I see where it is, remove this and remove setting it from prj.conf and add it to an nrf52840dk board overlay instead